clang/test/Sema/aarch64-sme2-intrinsics/acle_sme2_add_sub_za16.c

```diff
@@ -0,0 +1,29 @@
+// RUN: %clang_cc1 -triple aarch64-none-linux-gnu -target-feature +sme -verify -emit-llvm %s
```

Comment:
`-emit-llvm-only` is more conventional when the output is unneeded.

In our internal build system, PWD is read-only and `-emit-llvm` would cause a failure.
I changed this to `-emit-llvm-only` and changed `aarch64-none-linux-gnu` (none is sometimes used as an OS) to `aarch64` (generic ELF) to indicate that the behavior does not require Linux.

Reply:
Thanks!